jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/ui/activities/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.e_conomic.weatherapp.ui.activities
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import com.e_conomic.weatherapp.R
+import com.e_conomic.weatherapp.domain.commands.RequestForecastCommand
+import com.e_conomic.weatherapp.ui.ForecastListAdapter
+import kotlinx.android.synthetic.main.activity_main.*
+import org.jetbrains.anko.*
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        forecastList.layoutManager = LinearLayoutManager(this)
+
+        doAsync {
+            val forecastListResult = RequestForecastCommand("2618425").execute()
```

Comment:
Should mention that this is the id used for Copenhagen in the API. Will change it so it's clear for the reader. Some hard-coded stuff around at the moment.